src/go.mod

```diff
@@ -5,68 +5,63 @@ require (
 	github.com/BurntSushi/toml v0.3.1
```

Comment:
You never ran `go mod tidy`.

Reply:
Correct - I was waiting for the tests to pass. Doing that now.